study.md

```diff
@@ -103,7 +115,10 @@ Mollusca, and find the last record.
 | 2 | Stubby Squid | Rossia pacifica | Mollusca |
 
 ```ruby
-# your answer here
+# organism = Organism.find_by(common_name: 'Red Panda')
+organism = Organism.where(phylum: 'Mollusca').find_by(id: 2)
+(Can I just use something with '.last' above for finding my last mollusca organism?
+I want to do that, not sure how to, YET.)
 ```
```

Comment:
Yes! ActiveRecord has a .last method to retrieve the last record in a given set!